src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -126,7 +126,7 @@ internal enum MessageID
         IDS_FeatureTuples = MessageBase + 12711,
         IDS_FeatureOutVar = MessageBase + 12713,
 
-        IDS_FeatureIOperation = MessageBase + 12714,
+        // IDS_FeatureIOperation = MessageBase + 12714,
```

Comment:
are we keeping this around to enable it back for data flow analysis ?

Reply:
I used existing pattern in this file - I think they probably do it so we know about unused code. I am fine to delete this line if compiler team prefer that.